<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../resources/logo64x64.png</normaloff>../resources/logo64x64.png</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>24</width>
    <height>24</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_datos">
       <attribute name="title">
        <string>Vista de datos</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <layout class="QGridLayout" name="grid_datos">
          <item row="0" column="0">
           <widget class="QTableWidget" name="tableWidget"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_resultados">
       <attribute name="title">
        <string>Vista de comandos</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout">
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>20</number>
          </property>
          <item row="0" column="0" rowspan="5">
           <widget class="QGroupBox" name="groupBox_3">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="title">
             <string>Lista de variables</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_7">
             <item row="0" column="0">
              <widget class="QListWidget" name="list_variables">
               <property name="maximumSize">
                <size>
                 <width>300</width>
                 <height>16777215</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="1" rowspan="4" colspan="2">
           <widget class="QGroupBox" name="groupBox_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="title">
             <string>Resultados</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_6">
             <item row="0" column="0">
              <widget class="QTextBrowser" name="text_result">
               <property name="tabChangesFocus">
                <bool>false</bool>
               </property>
               <property name="undoRedoEnabled">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="1" colspan="2">
           <widget class="QGroupBox" name="groupBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>200</height>
             </size>
            </property>
            <property name="title">
             <string>Ventana de comandos</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_4">
             <item row="1" column="1">
              <widget class="QPushButton" name="button_ejecutar">
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Ejecutar</string>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QPushButton" name="button_clean">
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Limpiar ventana</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="0" column="0" colspan="3">
              <widget class="QTextEdit" name="edit_comandos">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>16777215</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="mnu_archivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="act_abrir_archivo"/>
    <addaction name="act_guardar_archivo"/>
    <addaction name="act_exportar_archivo"/>
    <addaction name="separator"/>
    <addaction name="act_Cerrar"/>
   </widget>
   <widget class="QMenu" name="mnu_datos">
    <property name="title">
     <string>Datos</string>
    </property>
    <widget class="QMenu" name="menuVariables">
     <property name="title">
      <string>Variables</string>
     </property>
     <widget class="QMenu" name="menuCrear_nueva">
      <property name="title">
       <string>Crear nueva</string>
      </property>
      <addaction name="act_newvar_1"/>
      <addaction name="act_newvar_2"/>
      <addaction name="act_newvar_3"/>
     </widget>
     <addaction name="menuCrear_nueva"/>
     <addaction name="act_discretizar"/>
     <addaction name="act_recodificar"/>
     <addaction name="act_tipificar"/>
     <addaction name="act_extraer_mues"/>
     <addaction name="actionTransformar"/>
    </widget>
    <addaction name="act_agregado"/>
    <addaction name="act_segmentado"/>
    <addaction name="menuVariables"/>
   </widget>
   <widget class="QMenu" name="mnu_analisis">
    <property name="title">
     <string>Análisis</string>
    </property>
    <addaction name="act_descriptiva"/>
    <addaction name="act_contraste"/>
    <addaction name="act_anovas"/>
    <addaction name="act_regresion"/>
    <addaction name="act_logistica"/>
    <addaction name="act_multivariado"/>
   </widget>
   <widget class="QMenu" name="mnu_graficos">
    <property name="title">
     <string>Gráficos</string>
    </property>
    <addaction name="act_graficos"/>
   </widget>
   <widget class="QMenu" name="mnu_acerca">
    <property name="title">
     <string>Acerca de</string>
    </property>
    <addaction name="act_sobre"/>
   </widget>
   <addaction name="mnu_archivo"/>
   <addaction name="mnu_datos"/>
   <addaction name="mnu_analisis"/>
   <addaction name="mnu_graficos"/>
   <addaction name="mnu_acerca"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="act_abrir">
   <property name="text">
    <string>Abrir archivo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="act_guardar">
   <property name="text">
    <string>Guardar archivo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="act_exportar">
   <property name="text">
    <string>Exportar</string>
   </property>
  </action>
  <action name="act_agregado">
   <property name="text">
    <string>Agregado</string>
   </property>
  </action>
  <action name="act_segmentado">
   <property name="text">
    <string>Segmentado</string>
   </property>
  </action>
  <action name="act_union">
   <property name="text">
    <string>Union</string>
   </property>
  </action>
  <action name="act_descriptiva">
   <property name="text">
    <string>Estadistica descriptiva</string>
   </property>
  </action>
  <action name="act_contraste">
   <property name="text">
    <string>Contraste de medias</string>
   </property>
  </action>
  <action name="act_anovas">
   <property name="text">
    <string>ANOVA</string>
   </property>
  </action>
  <action name="act_regresion">
   <property name="text">
    <string>Regresión múltiple</string>
   </property>
  </action>
  <action name="act_logistica">
   <property name="text">
    <string>Logística</string>
   </property>
  </action>
  <action name="act_multivariado">
   <property name="text">
    <string>Análisis multivariado</string>
   </property>
  </action>
  <action name="act_sobre">
   <property name="text">
    <string>Sobre ULLRToolbox</string>
   </property>
  </action>
  <action name="act_graficos">
   <property name="text">
    <string>Añadir gráficos</string>
   </property>
  </action>
  <action name="act_cerrar">
   <property name="text">
    <string>Cerrar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="act_discretizar">
   <property name="text">
    <string>Discretizar</string>
   </property>
  </action>
  <action name="act_ordenar_var">
   <property name="text">
    <string>Ordenar</string>
   </property>
  </action>
  <action name="act_recodificar">
   <property name="text">
    <string>Recodificar</string>
   </property>
  </action>
  <action name="act_tipificar">
   <property name="text">
    <string>Tipificar</string>
   </property>
  </action>
  <action name="act_extraer_mues">
   <property name="text">
    <string>Extraer muestra</string>
   </property>
  </action>
  <action name="act_newvar_1">
   <property name="text">
    <string>Variables existentes</string>
   </property>
  </action>
  <action name="act_newvar_2">
   <property name="text">
    <string>Funciones de resumen</string>
   </property>
  </action>
  <action name="act_newvar_3">
   <property name="text">
    <string>Algoritmo</string>
   </property>
  </action>
  <action name="actionTransformar">
   <property name="text">
    <string>Transformar</string>
   </property>
  </action>
  <action name="act_abrir_archivo">
   <property name="text">
    <string>Abrir archivo</string>
   </property>
  </action>
  <action name="act_guardar_archivo">
   <property name="text">
    <string>Guardar archivo</string>
   </property>
  </action>
  <action name="act_exportar_archivo">
   <property name="text">
    <string>Exportar archivo</string>
   </property>
  </action>
  <action name="act_Cerrar">
   <property name="text">
    <string>Cerrar</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>act_cerrar</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
